bin/MadGraph5_aMCatNLO/cards/production/13TeV/Singlet_Triplet_Higgs_portal_DM_ms200_mt220_ct3mm/Singlet_Triplet_Higgs_portal_DM_run_card.dat

```diff
@@ -0,0 +1,270 @@
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+ 1 = nevents ! Number of unweighted events requested 
+      0       = iseed   ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+        1     = lpp1    ! beam 1 type 
+        1     = lpp2    ! beam 2 type
+     6500     = ebeam1  ! beam 1 total energy in GeV
+     6500     = ebeam2  ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ nn23lo1    = pdlabel     ! PDF set                                     
+ 230000     = lhaid       ! if pdlabel=lhapdf, this is the lhapdf number
+
+#*********************************************************************
+# Renormalization and factorization scales                           *
+#*********************************************************************
+ F        = fixed_ren_scale  ! if .true. use fixed ren scale
+ F        = fixed_fac_scale  ! if .true. use fixed fac scale
+ 91.1880  = scale            ! fixed ren scale
+ 91.1880  = dsqrt_q2fact1    ! fixed fact scale for pdf1
+ 91.1880  = dsqrt_q2fact2    ! fixed fact scale for pdf2
+ 1        = scalefact        ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+ 0        = ickkw            ! 0 no matching, 1 MLM, 2 CKKW matching
+ 1        = highestmult      ! for ickkw=2, highest mult group
+ 1        = ktscheme         ! for ickkw=1, 1 Durham kT, 2 Pythia pTE
+ 1        = alpsfact         ! scale factor for QCD emission vx
+ F        = chcluster        ! cluster only according to channel diag
+ F        = pdfwgt           ! for ickkw=1, perform pdf reweighting
+ 5        = asrwgtflavor     ! highest quark flavor for a_s reweight
+ T        = clusinfo         ! include clustering tag in output
+ 3.0      = lhe_version       ! Change the way clustering information pass to shower.        
+ 0.0	  = time_of_flight   ! Time of Flight information added
```

Comment:
Your sample name indicates your particles fly 3mm but time_of_flight is set to 0?
Would this not be something you want?

@agrohsje Correct me if I am wrong, I have never used this parameter.

Reply:
The time_of_flight parameter sets the threshold above which madgraph starts including the parameter value for the corresponding particles. Hence a threshold of 0.0 means all values of time of flight are recorded. Anyway, in order to include time_of_flight in the CMSSW grid pack creation, one needs to include the following PR. [1]

[1]  https://github.com/cms-sw/genproductions/pull/2623/files

Reply:
Oh I see thank you for the answers!